scripts/remove_after_use/reindex_quickfiles.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+import progressbar
+from django.core.paginator import Paginator
+
+from website.app import setup_django
+setup_django()
+
+from website.search.search import update_file
+from addons.osfstorage.models import OsfStorageFile
+
+PAGE_SIZE = 100
+
+def reindex_quickfiles(dry):
+    qs = OsfStorageFile.objects.filter(node__type='osf.quickfilesnode')
```

Comment:
I left the `order_by` off in my comment, whoops. Still need that to make `Paginator` happy: 
```
UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'addons.osfstorage.models.OsfStorageFile'> IncludeQuerySet.
```